doc/robot_frames.rst

```diff
@@ -0,0 +1,61 @@
+Robot Frames
+============
+
+The URDF model uses multiple frames in the robot's base. This page describes why that is and what
+the purpose of each of that frames is. In particular, there are three frames representing the
+robot's base:
+
+- ``base_link`` - This serves as the root link if the kinematic chain. It follows `REP-103
+  <https://ros.org/reps/rep-0103.html>`_ , where "forward" for the ``base_link`` is defined as the
+  direction of the arm pointing to for an all-zero joint configuration.
+
+  .. figure:: frames/base_link.png
+     :alt: The robot with an all-zeros joint configuration showing its base_link frame
+
+     The robot with an all-zeros joint configuration showing its base_link frame
+
+- ``base`` - This is the frame that is used by the robot controller to represent the robot's base.
```

Comment:
Does *base* refer to the physical base (ie: the link) or the `Base` frame?

The idea (of the REP in any case) would be the latter (ie: `Base` coordinate frame).